packages/flutter/lib/src/services/system_navigator.dart

```diff
@@ -35,4 +35,33 @@ class SystemNavigator {
   static Future<void> pop({bool? animated}) async {
     await SystemChannels.platform.invokeMethod<void>('SystemNavigator.pop', animated);
   }
+
+  /// Notifies the platform for a route information change.
+  ///
+  /// On Web, creates a new browser history entry and update URL with the route
+  /// information stores in the [args].
+  static void routeInformationUpdated(Object args) {
```

Comment:
This shouldn't be typed "Object". Maybe take everything that's in the map as named arguments?